docs/threat-model.md

```diff
@@ -0,0 +1,54 @@
+# Threat Model
+
+## Overview
+
+netperf is a framework and set of tools for measuring network (and related components) performance. It is designed to be used in a controlled environment, with full control over the systems involved in the test. The primary goal of netperf is to provide accurate and repeatable performance measurements.
+
+### Components
+
+#### netperf Repository
+
+The netperf repository contains the scripts and yaml for the netperf framework and tools. It also contains various secrets to access an Azure Subscription.
+
+#### GitHub Actions
+
+GitHub Actions is used to run the netperf tests. They use the the yaml, scripts and secrets from the netperf repository to run the jobs necessary to perform the tests. These actions are configured to (securely) remotely triggered via [workflow_call](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_call), (securely) manually via [workflow_dispatch](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch), or via a netperf Pull Request. Pull Request automation from outside contributors **requires explicit approval**.
```

Comment:
That's more of a question/problem for the other repositories, right? Since we also own those, I agree it's important, but do we really need anything in this document? If so, would you propose some specific text?

Reply:
PAT and VM password are secrets that should be rotated.